spec/scrabble-class_spec.rb

```diff
@@ -0,0 +1,60 @@
+require './lib/scrabble/scrabble-class.rb'
+
+describe "Scrabble" do
+  describe "score(word)" do
+    context "for valid word input" do
+      it "returns 7 for'aaaaaaa'" do
+        expect(Scrabble::Scrabble.score("aaaaaaa")).to eq(7)
+      end
+    end
+
+    context "for invalid word input" do
+      it "returns 'ERROR' for numbers in input" do
+        expect(Scrabble::Scrabble.score("hel4lo")).to eq("ERROR")
+      end
+
+      it "returns 'ERROR' for spaces in input" do
+        expect(Scrabble::Scrabble.score("he llo")).to eq("ERROR")
+      end
+
+      it "returns 'ERROR' for nothing in input" do
+        expect(Scrabble::Scrabble.score("")).to eq("ERROR")
+      end
+
+      it "returns 'ERROR' for nil in input" do
+        expect(Scrabble::Scrabble.score(nil)).to eq("ERROR")
+      end
+
+      it "returns 'ERROR' if length of the word is > 7" do
+        expect(Scrabble::Scrabble.score("eightlet")).to eq("ERROR")
+      end
+    end
+  end
+
+  describe "highest_score_from(array_of_words)" do
+    it "returns word with highest score" do
```

Comment:
You've defined a bunch of words in your specs, and as a reader I have no idea what the values would be. So if you added comments like the ones you have [here](https://github.com/corinnepingul/C3Projects--Scrabble/blob/crp/master/spec/player-class_spec.rb#L43), it's be helpful